src/main/java/hydroguard/api/mock/models/reocurringTask/ReoccurringTaskDTO.java

```diff
@@ -0,0 +1,92 @@
+package hydroguard.api.mock.models.reocurringTask;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class ReoccurringTaskDTO {
+    private UUID id;
+    private UUID userId;
+    private UUID plantId;
+    private String description;
+    private boolean completed;
+    private LocalDateTime dueAt;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+    private int intervalSeconds;
```

Comment:
Deze staat niet in de constructor, klopt dat? Ook niet in de get setters. Volgensmij komt hij daarom niet in de swagger tevoorschijn.